api/samples/neoncounter.c

```diff
@@ -0,0 +1,703 @@
+/* ******************************************************************************
+ * Copyright (c) 2014-2018 Google, Inc.  All rights reserved.
```

Comment:
The header here seems wrong. But I am not sure what to put there.

Reply:
This comes directly from the inscount.cpp client